src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -12,6 +12,33 @@ const generateInitialOptions = (meta) => {
   return {
     reverse: higherIsBetter,
     scaleLabel: higherIsBetter ? 'Score' : 'Load time',
+    tooltips: {
+      callbacks: {
+        footer: (tooltipItems, data) => {
+          let delta = 0;
+          tooltipItems.map((tooltipItem) => {
+            // console.log(data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].y,
+            // ' current');
+            // console.log(data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index - 1].y,
+            // 'previous');
+
+            // console.log(data.datasets, 'Data.DataSet');
+            // console.log(tooltipItem.index, 'tooltipitem.index');
+            // console.log(tooltipItem.index - 1, 'previous index');
+            // condition, if != 0 ?
```

Comment:
I realize this is a work in progress since there are more data points to add, but it makes it easier for the reviewer to have less comments/clutter when reviewing.